code/datums/wires/syndicatebomb.dm

```diff
@@ -42,9 +46,12 @@
 				playsound(B, 'sound/machines/click.ogg', 30, 1)
 				B.active = TRUE
 				B.update_icon()
+			if(B.delayedlittle)
+				holder.visible_message("<span class='notice'>\icon[B] Nothing happens.</span>")
 			else
 				holder.visible_message("<span class='notice'>\icon[B] The bomb seems to hesitate for a moment.</span>")
-				B.timer += 5
+				B.timer += 10
+				B.delayedlittle = TRUE

```

Comment:
With the current code, if I pulse this wire on a bomb that is neither active or defused, I will get both "the bomb start ticking!" and " the bomb seems to hesitate".

the logic should be "if(!B.active && !B.defused) XXXX else if(B.delaylittle) XXXX else XXXXX